homeassistant/components/soundtouch/manifest.json

```diff
@@ -2,6 +2,7 @@
   "domain": "soundtouch",
   "name": "Bose Soundtouch",
   "documentation": "https://www.home-assistant.io/integrations/soundtouch",
-  "requirements": ["libsoundtouch==0.7.2"],
+  "requirements": ["libsoundtouch==0.8"],
+  "dependencies": [],
```

Comment:
```suggestion
```

Reply:
I'm pretty new, especially to tests. Do you mean something like the test_playing_media test
https://github.com/home-assistant/core/blob/9b127403467e2aee4da8209c92a7c577609f9c29/tests/components/soundtouch/test_media_player.py#L326-L328

Reply:
Yes, exactly. :+1:

Reply:
I don't fully understand what I did, but I think I adapted as it should